frontend/src/components/regions/CardDetails/CardDetails.test.js

```diff
@@ -111,3 +111,36 @@ test(`canSetDueTime function returns false if card doesn't belong to current use
 
   expect(canSetDueTime({ card, viewedUser, authUser })).toBe(false);
 });
+
+test(`canSetDueTime function returns true if card doesn't belongs to current user, 
+        current user has management permissions and is a super user and card has no due time set`, () => {
+  const viewedUser = {
+    id: 295,
+    teamMemberships: {
+      28: {
+        id: 28,
+        name: "Cohort 22 web dev",
+      },
+    },
+  };
+  const authUser = {
+    userId: 200,
+    permissions: {
+      teams: {
+        28: {
+          id: 28,
+          name: "Cohort 22 web dev",
+          active: true,
+          isSuperuser: true,
```

Comment:
This data structure doesn't match the structure we use on Tilde. And most of the data is not relevant to the test. 
Either a person is a superuser or they aren't. A person doesn't get superuser permission over a single team.

Please open Tilde in your browser. Open the redux dev tools (chrome) and look at the state. Look inside `state.App.authUser`.

Please use a data structure that matches that. 

also for this test: `authUser.permissions = {}`.  No individual permissions are necessary if a person has superuser status.